### PR TITLE
Bus Monitor - Ability to watch values from more than one bus at a time

### PR DESCRIPTION
**Pull request by robertsaj**

### CODE REVIEW

File: electron/main.js [hunk 1]
```diff
@@ -73,7 +75,7 @@ app.on('ready', createWindow);
 // Quit when all windows are closed.
 app.on('window-all-closed', function () {
   // On OS X it is common for applications and their menu bar
-  // to stay active until the user quits explicitly with Cmd + Q
+  // to stay state until the user quits explicitly with Cmd + Q
```
Comment (hunterparks) [inline electron/main.js]: What does this mean?

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -9,14 +9,14 @@ const routes: Routes = [
   {
     path: '',
     pathMatch: 'full',
-    redirectTo: 'editor'
+    redirectTo: 'simulator'
```
Comment (hunterparks) [inline src/app/app.routes.ts]: By default we want the user pushed to the simulator view?

File: electron/main.js (hunk 1 above)
Comment (robertsaj) [reply electron/main.js]: That's a prime example of refactoring thinking it's really good at its job. Let me know if you find more, I'll fix them

File: src/app/components/common/tooltip/tooltip-container/tooltip-container.component.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export class TooltipContainerComponent {
 
   public tooltips: any[];
 
-  constructor(tooltipService: TooltipService) {
+  constructor (tooltipService: TooltipService) {
```
Comment (hunterparks) [inline src/app/components/common/tooltip/tooltip-container/tooltip-container.component.ts]: Remove space after the word constructor.

File: src/app/components/pages/simulator-view/simulator-view.component.ts [hunk 4]
```diff
@@ -32,33 +32,23 @@ export class SimulatorViewComponent {
...
-    let messageObject: any = JSON.parse(message);
-    Object.keys(messageObject).map((key: string) => {
-      let selectedBus: Bus = this.simulator.bus.find((bus: Bus) => {
-        return bus.name.toLowerCase() === key;
-      });
-      if (selectedBus) {
-        selectedBus.data.next(InspectService.formatTooltip(selectedBus, messageObject[key].state));
-      }
-    });
+  constructor (
```
Comment (hunterparks) [inline src/app/components/pages/simulator-view/simulator-view.component.ts]: Remove space after word constructor.

File: src/app/components/simulator/bus/bus/bus.component.ts [hunk 5]
```diff
@@ -20,7 +20,7 @@ export class BusComponent implements OnInit {
   public arrows: string[] = [];
   // Input string with a list of coordinates
   @Input('svg-bus') public bus: Bus = null;
-  // The active color of the bus
+  // The state color of the bus
```
Comment (hunterparks) [inline src/app/components/simulator/bus/bus/bus.component.ts]: I think this is supposed to be "active" again.

File: src/app/services/simulator/websocket/websocket.service.ts [hunk 6]
```diff
@@ -11,7 +11,7 @@ export class WebsocketService {
   /**
    * Simple constructor for the service, ensures all instance variables are initialized and connects the socket
    */
-  constructor() {
+  constructor () {
```
Comment (hunterparks) [inline src/app/services/simulator/websocket/websocket.service.ts]: Remove space after word constructor.

Comment (robertsaj) [reply src/app/services/simulator/websocket/websocket.service.ts]: All of the spaces are removed.

**Review by stoehraj — APPROVED** [review]

**Review by hunterparks — APPROVED** [review]
Everything looks good. I did notice that a few components have the functions formatTooltip, initialize, and inspect left as TODO items - as long as those will be addressed in a future PR, I'm happy with these changes.

**Comment by hunterparks** [comment]
Note: There are some linting issues, but those can stay for now.

**Comment by robertsaj** [comment]
There is a task in Trello for linting. I was planning on just sitting down one day and cleaning it all up at once.

The todos are in there for the various simulator components because I'll need to talk with Curt and Larry about which components have which behaviors and how they should be implemented. They'll need to be added at some point, but they're not required for the bus monitor.

**Review by skusel — APPROVED** [review]

**Review by curthenrichs — APPROVED** [review]

**Pull request merged**